graphene/src/box_.rs

```diff
@@ -41,35 +33,35 @@ impl Box {
 
     #[doc(alias = "graphene_box_init_from_box")]
     #[doc(alias = "new_from_box")]
-    pub fn from_box(src: &Box) -> Box {
+    pub fn from_box(src: &Box) -> Self {
         assert_initialized_main_thread!();
         unsafe {
-            let mut b = Box::uninitialized();
+            let mut b = Self::uninitialized();
             ffi::graphene_box_init_from_box(b.to_glib_none_mut().0, src.to_glib_none().0);
             b
         }
     }
 
     #[doc(alias = "graphene_box_init_from_points")]
     #[doc(alias = "new_from_points")]
-    pub fn from_points(&mut self, points: &[Point3D]) -> Box {
+    pub fn from_points(&mut self, points: &[Point3D]) -> Self {
```

Comment:
Erm this one is wrong

```suggestion
    pub fn from_points(points: &[Point3D]) -> Self {
```

`self` was not even used.